.eslintrc.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-  extends: 'frost-standard'
+  extends: 'frost-standard',
```

Comment:
Don't want to leave the same comment as in the other repo?

Reply:
Thanks for catching that. I will add that in.